packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -369,7 +369,8 @@ async function createOutgoingPayment(
             client: options.client,
             metadata: options.metadata,
             state: OutgoingPaymentState.Funding,
-            grantId
+            grantId,
+            createdAt: new Date()
```

Comment:
This is done automatically

(we have `table.timestamp('createdAt').defaultTo(knex.fn.now())` in the create outgoing payments table migration)